app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/BaseViewModel.kt

```diff
@@ -116,5 +116,5 @@ abstract class BaseViewModel : ViewModel(), KoinComponent {
     }
 
     // Ties flow to viewModelScope to give StateFlow.
-    fun <T> Flow<T>.groundState(initialValue: T) = this.stateIn(viewModelScope, SharingStarted.Lazily, initialValue)
+    fun <T> Flow<T>.groundState(initialValue: T) = this.stateIn(viewModelScope, SharingStarted.Eagerly, initialValue)
```

Comment:
@BottleRocket-Colin I'm curious to hear the reasoning for moving to Eagerly from Lazily. Was there a particular use case that works better when using Eagerly?

Reply:
Lazily was causing some timing issues during testing..

Reply:
@BottleRocket-Colin Since making this change could affect our direction for quite a few projects, I'd love to have a deeper understanding of the kinds of timing issues you've seen so that we can share that info more broadly :)

Reply:
The main behavioral change is that this intermediate StateFlow will start collecting from the original flow upon instantiation instead of waiting until it is collected from.  This way events that occur before final collection should not get dropped.